index.html

```diff
@@ -1,72 +1,225 @@
+                <ul class="list-unstyled">
+                    <li>Unlimited Documents</li>
+                    <li>Unlimited Reviews</li>
+                    <li>Unlimited Members</li>
+                    <li>Unlimited Support</li>
+                </ul> -->
+                <a href="https://sgrewritten.org/codestyle" class="btn my-4 font-weight-bold atlas-cta cta-ghost">Join our Discord</a>
+            </div>
+            <div data-aos="fade-right" data-aos-delay="200" data-aos-duration="1000" data-aos-once="true" class="col-md-4 text-center py-4 mt-5">
+                <h4 class="my-4">Sponsors</h4>
+               <!-- <p class="font-weight-bold">$ <span class="display-2 font-weight-bold">0</span> / MO.</p>
+                <ul class="list-unstyled">
+                    <li>Up to 5 Documents</li>
+                    <li>Up to 3 Reviews</li>
+                    <li>5 team Members</li>
+                    <li>Limited Support</li>
+                </ul> -->
+                <a href="https://sgrewritten.org/sponsor" class="btn my-4 font-weight-bold atlas-cta cta-ghost">Sponsor</a>
+            </div>
+            <div data-aos="fade-left" data-aos-delay="200" data-aos-duration="1000" data-aos-once="true" class="col-md-4 text-center py-4 mt-5">
+                <h4 class="my-4">Alpha Testers</h4>
+                <!-- <p class="font-weight-bold">$ <span class="display-2 font-weight-bold">0</span> / MO.</p>
+                <ul class="list-unstyled">
+                    <li>Up to 15 Documents</li>
+                    <li>Up to 10 Reviews</li>
+                    <li>25 team Members</li>
+                    <li>Limited Support</li>
+                </ul> -->
+                <a href="https://discord.com/channels/853659042799026177/853659045671337996/978737916782211162" class="btn my-4 font-weight-bold atlas-cta cta-ghost">Join our Discord</a>
+            </div>
+        </div>
+    </div>
+    <!-- contact -->
+    <div class="jumbotron jumbotron-fluid" id="contact" style="background-image: url(img/bg.jpg);">
+        <div class="container my-5">
+            <div class="row justify-content-between">
+                <div class="col-md-6 text-white">
+                    <h2 class="font-weight-bold">Contact Us</h2>
+                    <p class="my-4">
+                        "We got a message from the Tok'ra. Apparently they've been trying to contact us for over three months." <br>          
+                        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; - Maj. Carter, Stargate S4E6
+                    </p>
+                    <ul class="list-unstyled">
+                        <li>Email : contact@sgrewritten.org</li>
+                        <li>Address : Scandinavia, Mostly.</li>
+                    </ul>
+                </div>
+                <div class="col-md-6">
+                    <form>
+                        <div class="row">

```

Comment:
Do we need a contact form? Currently, we have just been using a discord link and an email.
If we do keep one, it means we technically need to include a privacy policy page.

Reply:
Uhm, yeah that would be a... pain in the butt to do. However we can use it for credits. :D